dev/com.ibm.ws.wsbytebuffer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2011, 2020 IBM Corporation and others.
```

Comment:
change date

Reply:
Ask Tom about if 'moved' messages need new id/review/translation etc.

Reply:
Speak to Phil Nicoll when this sort of thing needs reviewed. He is aware of this PR/discussed.